deephub/trainer/exporters.py

```diff
@@ -0,0 +1,91 @@
+# Source: https://github.com/bluecamel/best_checkpoint_copier
+import glob
+import os
+import shutil
+import tensorflow as tf
+
+class Checkpoint(object):
+  dir = None
+  file = None
+  score = None
+  path = None
+
+  def __init__(self, path, score):
+    self.dir = os.path.dirname(path)
+    self.file = os.path.basename(path)
+    self.score = score
+    self.path = path
+
+
+class BestCheckpointCopier(tf.estimator.Exporter):
+  checkpoints = None
+  checkpoints_to_keep = None
+  compare_fn = None
+  name = None
+  score_metric = None
+  sort_key_fn = None
+  sort_reverse = None
+
+  def __init__(self, name='best_checkpoints', checkpoints_to_keep=5, score_metric='Loss/total_loss',
+               compare_fn=lambda x,y: x.score < y.score, sort_key_fn=lambda x: x.score, sort_reverse=False):
```

Comment:
I would remove this function definition from the __init__ signature. It would be better to be a small function out of here. WDYT @nanton96 ?